src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -22,4 +22,8 @@ public User get(String login) {
         }
         return null;
     }
+
+    public void clear() {
+        Storage.people.clear();
+    }
```

Comment:
The `clear` method is a utility method that seems to be used for tests. Ensure that it's not accessible or used in production code as it can lead to data loss.

Reply:
It's bad idea to change code only for testing purpose.